pkg/reconciler/apis/apibinding/apibinding_controller.go

```diff
@@ -228,15 +228,16 @@ func (c *controller) enqueueAPIExport(obj interface{}) {
 		return
 	}
 
-	klog.V(2).Infof("Mapping APIExport %q", key)
 	bindingsForExport, err := c.apiBindingsIndexer.ByIndex(indexAPIBindingsByWorkspaceExport, key)
 	if err != nil {
 		runtime.HandleError(err)
 		return
 	}
 
-	for _, apiBinding := range bindingsForExport {
-		c.enqueueAPIBinding(apiBinding)
+	for _, obj := range bindingsForExport {
+		apiBinding := obj.(*apisv1alpha1.APIBinding)
+		klog.V(2).Infof("Queueing APIBinding %s|%s because of APIExport %s|%s", logicalcluster.From(apiBinding), apiBinding.Name, key)
```

Comment:
Mismatch between % signs & number of variable args. Do you want to convert key to cluster|name?

Reply:
done